src/vmspawn/vmspawn.c

```diff
@@ -804,6 +811,11 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return r;
 
+        if (!arg_quiet)
+                log_info("Spawning VM %s on %s.\n"
+                         "Press Ctrl-a q RET to kill VM.",
```

Comment:
I think this should be ctrl-a+x? for me ctrl-a+q doesn't kill the VM but ctrl-a+x does

Reply:
you have to hit RET after the ctr-a q...

but if ctrl-a x works too let's switch to that, it's shorter. (didn't know that works)